app/views/content/landing/return-to-teaching-advisers/_adviser.html.erb

```diff
@@ -3,12 +3,23 @@
     <section class="col col-720 col-space-0">
          <%= render Content::AdviserComponent.new(
         title: "Free one-to-one support by phone, text or email",
-        intro: "If you're thinking about returning to teaching in England, an adviser can help you get classroom experience, search for teaching jobs and support you with the application process. You can talk to an adviser as little or as often as you need. To be eligible for an adviser, you'll need English qualified teacher status (QTS) and want to
-  teach in an English state school.",
         color: "transparent",
         margin: false,
         heading: :l
-      ) %>
+      ) do %>
+
+        <p>
+          <p>If you're thinking about returning to teaching in England, an adviser can help you:</p>
+          <ul>
+          <li>get classroom experience</li>
+          <li>search for teaching jobs</li>
+          <li>support you with the application process.</li>
+          </ul>
+        <p>You can talk to an adviser as little or as often as you need.</p>
+        <p>To be eligible for an adviser, you'll need English qualified teacher status (QTS) and want to teach in an English state school.<p>
+        </p>
```

Comment:
line 20 is redundant - please delete

Reply:
line 19 will also need changing as the end `<p>` should be a `</p>`